be/src/exec/tablet_sink.cpp

```diff
@@ -330,13 +332,14 @@ Status NodeChannel::add_row(BlockRow& block_row, int64_t tablet_id) {
 }
 
 void NodeChannel::_sleep_if_memory_exceed() {
-    size_t begin_us = _mem_exceeded_block_ns / 1000;
+    static size_t times = 1;
```

Comment:
Should be a member variable? it's lifecycle should be consistent with `_mem_exceeded_block_ns`?